src/Trait/ApiListTrait.php

```diff
@@ -26,6 +27,13 @@ protected function getListObject()
         return null;
     }
 
+    protected function modifyResponseListObjects(array|Collection $results)
+    {
+        foreach ($results as $result) {
+            $this->modifyResponseObject($result);
+        }
+    }
+
```

Comment:
Should this be added to `ApiExportCsvTrait` and `ApiExportExcelTrait`? Does it make sense there as well?